app/views/layouts/application.html.slim

```diff
@@ -36,9 +36,7 @@ html
 
       .notifications.navbar-static-top#flash
         - if flash[:alert]
-          .alert.alert-warning.flash= flash[:alert]
-        - if flash[:login_link]
-          = render partial: "publishers/login_link"
+          .alert.alert-warning.flash== flash[:alert]
```

Comment:
Seems risky- have you audited to make sure nothing is putting user-submitted data into the flash string?

Reply:
Yes, I checked and only translation strings ever get put into `flash[:alert]`. As with other cases where we've used the html-safe `==`, this lets us lean on translation strings for simple formatting and links rather than using a one-off partials as was previously being done in this case.

Reply:
so originally I shared this concern of html_yolo, so that's why we created a separate login_link bit -- to isolate the area in which we're using html_safe

can we maybe make a flash[:alert_html_safe] in cases where we really want it?